src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -61,7 +61,8 @@ public Command parseCommand(String userInput) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, HelpCommand.MESSAGE_USAGE));
         }
 
-        final String commandWord = matcher.group("commandWord");
+        final String commandWordCaseSensitive = matcher.group("commandWord");
+        final String commandWord = commandWordCaseSensitive.toLowerCase();
```

Comment:
Perhaps renaming the variables can make it easier to understand what they do e.g. commandWordUserInput for the first variable and commandWordLowercase for the second variable